DC-SLES-minvm

```diff
@@ -1,11 +1,11 @@
 ## ---------------------------- 
```

Comment:
If I recall corrctly, due to SEO reasons, the DC files and file names should be kind of "speaking" for itself. Maybe use `DC-SLES-minimal-vm`? That would need to be applied also to IDs. Certainly Stefan knows more about this.